src/test_workflow/compare_benchmark/compare_args.py

```diff
@@ -0,0 +1,41 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import argparse
+
+
+class CompareArgs:
+    id1: str
+    id2: str
+    results_format: str
+    results_numbers_align: str
+    results_file: str
+    show_in_results: str
+
+    def __init__(self) -> None:
+        self.parser = argparse.ArgumentParser(description="Compare two IDs")
+        self.parser.add_argument("id1", type=str, help="The baseline ID to compare")
+        self.parser.add_argument("id2", type=str, help="The contender ID to compare")
```

Comment:
Can we update the README file of test_workflow to know how exactly this feature work?